Apps/W1/HybridGP/app/src/Migration/Support/PurchaseOrders/GPPOMigrator.codeunit.al

```diff
@@ -128,7 +132,7 @@ codeunit 40108 "GP PO Migrator"
 
         repeat
             PurchaseLine.Init();
-            PurchaseLine."Document No." := GPPOP10110.PONUMBER;
+            PurchaseLine."Document No." := CopyStr(GPPOP10110.PONUMBER.Trim(), 1, MaxStrLen(PurchaseLine."Document No."));
```

Comment:
Will this lead to PO Number truncation or is it just to remove blank spaces?

Reply:
I only added this to get rid of the warning. This will not result in truncation, Document No. is larger than PONUMBER.